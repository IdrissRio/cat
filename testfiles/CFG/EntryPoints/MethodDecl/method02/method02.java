public class method02 {
  void m(int e, int g) { System.out.println(e + g); }
}

/**
 * Testing MethodDecl with parameters as entry point for the analysis
 */