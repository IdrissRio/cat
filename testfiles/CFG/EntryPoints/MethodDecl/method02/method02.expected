NodeName: Entry[15] Successors: [Exit[16]]
NodeName: Exit[16] Successors: []
NodeName: Entry[7] Successors: [Exit[8]]
NodeName: Exit[8] Successors: []
NodeName: Entry[29] Successors: [TypeAccess[49]]
NodeName: TypeAccess[49] Successors: [VarAccess[50]]
NodeName: VarAccess[50] Successors: [VarAccess[55]]
NodeName: VarAccess[55] Successors: [VarAccess[56]]
NodeName: VarAccess[56] Successors: [AddExpr[54]]
NodeName: AddExpr[54] Successors: [MethodAccess[51]]
NodeName: MethodAccess[51] Successors: [Exit[44]]
NodeName: Exit[44] Successors: []

NodeName: Exit[16] Predecessor: [Entry[15]]
NodeName: Entry[15] Predecessor: []
NodeName: Exit[8] Predecessor: [Entry[7]]
NodeName: Entry[7] Predecessor: []
NodeName: Exit[44] Predecessor: [MethodAccess[51]]
NodeName: MethodAccess[51] Predecessor: [AddExpr[54]]
NodeName: AddExpr[54] Predecessor: [VarAccess[56]]
NodeName: VarAccess[56] Predecessor: [VarAccess[55]]
NodeName: VarAccess[55] Predecessor: [VarAccess[50]]
NodeName: VarAccess[50] Predecessor: [TypeAccess[49]]
NodeName: TypeAccess[49] Predecessor: [Entry[29]]
NodeName: Entry[29] Predecessor: []
